### ISSUE TITLE
Daily name registration 7/11-7/14, 7/18-7/21

### ISSUE THREAD

**Issue by pstan26** [blockstack/blockstack-browser#611]
This week, each member of the core team is actively registering new names for ourselves daily to test registration.

Each day, we'll add our newly registered name in a comment below, along with a description of errors experienced and the total cost. We will refund your expenses at the end of the week.

We've done this before here: https://github.com/blockstack/blockstack-browser/issues/555#issuecomment-312664082

**Comment by shea256** [comment]
@kantai @larrysalibra Are we ready to start doing this?

**Comment by kantai** [comment]
I'm registering right now, but I think we want a new build of the dmg before everyone begins testing.

**Comment by kantai** [comment]
https://explorer.blockstack.org/name/ablankst_jul11_unsafe.id

Registered! 0-conf payment on the pre-order, unsafe mode + browser owner key used for each of the three required operations, total cost was ~0.004 btc

**Comment by larrysalibra** [comment]
> @kantai @larrysalibra Are we ready to start doing this?

No, we aren't.

**Comment by larrysalibra** [comment]
We're ready now! Registered a name with https://github.com/blockstack/blockstack-browser/releases/tag/v0.11.0

Details: 
https://explorer.blockstack.org/name/zhongnanhai.id

**Started at:** 2017-07-12 21:03:50 (wow! the pre order transaction was mined within ~90 seconds of clicking register! https://explorer.blockstack.org/nameops/475468)
**Victory declared at:** 2017-07-12 22:08:22
**Total cost:** ~0.005

**Comment by pstan26** [comment]
Great -- thanks Larry

**Comment by kantai** [comment]
Today's name registration, from Larry's v0.11 build.

https://explorer.blockstack.org/name/res_publica.id

**Comment by larrysalibra** [comment]
Today's name registration with v0.11.1 build:

https://explorer.blockstack.org/name/waterpolo.id

Cost: ~0.005

I only had about ~0.0055 BTC in the wallet to begin with...after registration had 0.000587 BTC. Great job at fee estimation!

**Comment by kantai** [comment]
Yesterday's name registration: 

https://core.blockstack.org/v2/users/ablankst_testing_july17.id

I had a bunch of dust in my wallet (actually about ~0.004 btc worth of dust), so the registration was unpleasant.

(1) On the mac mini, ECDSA signs take a while, and it was doing multisig for every payment input it attempted to use while trying to determine the price of a name. This led to name price calculation times of ~30 seconds. blockstack-core has an issue created for this (and I will address this today).

(2) I tried to send the minimum amount to get to a name registration, but of course, this was wrong, because of the dust UTXOs, I needed to actually send more BTC. I'm not sure what to do about this problem? Perhaps "balance" could have an option that returns the sum of spendable inputs?

(3) However, this led to a different problem, because the balance that browser thought I had was enough to cover the cost of the registration process, browser allowed me to submit a registration to core, which failed because I didn't have enough funds. The error displayed in browser was just "There was a problem with your registration." -- core replies with a more verbose error message here, and the browser can indicate to the user that more BTC is required.
